packages/backend-modules/republik-crowdfundings/lib/payments/stripe/deleteCustomer.js

```diff
@@ -11,13 +11,16 @@ module.exports = async ({ userId, pgdb }) => {
     return
   }
 
-  for (let customer of customers) {
+  for (const customer of customers) {
     const account = accounts.find((a) => a.company.id === customer.companyId)
     if (account) {
       await account.stripe.customers.del(customer.id)
       await pgdb.public.stripeCustomers.deleteOne({
         id: customer.id,
       })
+      await pgdb.payments.stripeCustomers.deleteOne({
+        id: customer.id,
```

Comment:
this should be customerId: customer.id

Reply:
but in the end we should just get rid of the new stripe table. When I drafted the payments module I would it would be simpler to kept in the same db schema but I forgot that the old order flow also needs them.